### ISSUE TITLE
ft_full ?

### ISSUE THREAD

**Issue by xarryon** [ucbdrive/few-shot-object-detection#137]
Does the 'full' means more iteration will be used during training compared with ft?

**Comment by thomasehuang** [comment]
No, `ft-full` means fine-tuning the entire model rather than just the final layer.
